services/apps/alcs/src/alcs/application/application.controller.ts

```diff
@@ -122,6 +128,31 @@ export class ApplicationController {
   @Post('/:fileNumber/cancel')
   @UserRoles(...ROLES_ALLOWED_APPLICATIONS)
   async cancel(@Param('fileNumber') fileNumber): Promise<void> {
+    const applicationSubmission = await this.applicationSubmissionService.get(
```

Comment:
The block of loading the submission finding the primary contact, then loading the submission government seems pretty common. Should there be a SubmissionStatusEmailService that DRYS this up?

Reply:
Updated!